WasatchNET/EEPROM.cs

```diff
@@ -1552,7 +1558,7 @@ public virtual async Task<bool> readAsync()
                 slitSizeUM = ParseData.toUInt16(pages[0], 41);
 
                 startupIntegrationTimeMS = ParseData.toUInt16(pages[0], 43);
-                startupDetectorTemperatureDegC = ParseData.toInt16(pages[0], 45);
+                TECSetpoint = ParseData.toInt16(pages[0], 45);

```

Comment:
We should probably comment somewhere that we are assuming that any given spectrometer will only have ONE controllable TEC, and that on some models it may be the detector (Hamamatsu -R and -C models), and on others it may be the laser (XS SML models). Currently there is no support for a spectrometer on which both the detector and laser have independently configurable TECs. (Although some models have "locked-in" non-configurable TECs for one or the other component, like X/XM-R/C-ILP/C units have configurable detector TEC but locked-in laser TEC.)

Reply:
adding a comment here shortly

Reply:
comment added on line 236